util/annotations/helpers_test.go

```diff
@@ -110,6 +110,23 @@ func TestAddAnnotations(t *testing.T) {
 			},
 			changed: true,
 		},
+		{
+			name: "should return true if annotations are changed and have been nil before",
```

Comment:
is this test failing without your change?

Reply:
yes